cartographer_grpc/framework/service.cc

```diff
@@ -52,7 +52,7 @@ void Service::StartServing(
 void Service::StopServing() { shutting_down_ = true; }
 
 void Service::HandleEvent(Rpc::Event event, Rpc* rpc, bool ok) {
-  rpc->GetRpcEvent(event)->pending = false;
+  rpc->UpdateRpcEventState(event, false);
```

Comment:
naming: SetRpcEventState or better SetEventState (this is obviously about the Rpc)